src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -418,6 +426,9 @@ export class ChallengesubmitComponent implements OnInit {
     FORM_DATA.append('method_description', self.globalService.formValueForLabel(self.components, 'method_description'));
     FORM_DATA.append('project_url', self.globalService.formValueForLabel(self.components, 'project_url'));
     FORM_DATA.append('publication_url', self.globalService.formValueForLabel(self.components, 'publication_url'));
+    if (self.validFileUrl === true) {
+      FORM_DATA.append('file_url', self.globalService.formValueForLabel(self.components, 'file_url'));
```

Comment:
You need to send `file_url` or `input_file` depending on the selection of submission option not both, please check this case.